docs/basics/ScriptingwithR.md

```diff
@@ -0,0 +1,43 @@
+# Scripting with R
+Since most ecologists and biologists are more used to using R than shell, we provide here an R script to run *AP.exe*
+
```

Comment:
We should explain that using R like this, where we invoke _AP.exe_ with `system2` is identical to using a shell.

R interactive is just another shell after all, and RStudio is just another terminal.

So just using R to invoke AP.exe is adding a lot of complexity. The main reason to do this is if you want to integrate an analysis in with the rest of your R analysis, or to batch analyze data.